### ISSUE TITLE
Increased limit for belt input

### ISSUE THREAD

**Issue by drvic10k** [hetima/DSP_LSTM#23]
My use case:

I have a logistics station, that provides hydrogen, I also produce hydrogen locally. I supply the locally produced hydrogen to the station and I want to cover the missing amount with demand.
But when the demand is fulfilled by imports, the belt is blocked. I think the local demand delay is supposed to solve this problem, byt the 1% buffer is not enough to keep the belt running.

My ideal solution would be to have different amount for demand and different for storage, something like if you have demand level fulfilled, you can still manually drop more items in the station, but for the belt input.

**Comment by hetima** [comment]
I have also come across situations where 1% buffer is not enough.
Increasing the buffer is the easiest way to improve the situation, so I would like to consider it.
I am reluctant to use other methods because I think they would require complex implementation.

**Comment by drvic10k** [comment]
yes, custom buffer would be also fine, best would be per slot, but at least per station
